deploy_docker/000-default.conf

```diff
@@ -0,0 +1,29 @@
+<VirtualHost *:80>
+	# The ServerName directive sets the request scheme, hostname and port that
+	# the server uses to identify itself. This is used when creating
+	# redirection URLs. In the context of virtual hosts, the ServerName
+	# specifies what hostname must appear in the request's Host: header to
+	# match this virtual host. For the default virtual host (this file) this
+	# value is not decisive as it is used as a last resort host regardless.
+	# However, you must set it for any further virtual host explicitly.
+	#ServerName www.example.com
+	ServerName SimpleUI
+	ServerAdmin VionxService@vionx.com
```

Comment:
We should change in the future to a Largo Email address. Okay for now.

Reply:
Ideally we can get rid of these apache configs entirely relatively soon and just have node serve everything